tests/pytorch/mock_sparse/test_elementwise_op_sp.py

```diff
@@ -70,8 +102,10 @@ def test_expose_op():
     dgl.mock_sparse.mul(A, A)
     dgl.mock_sparse.div(A, A)
 
-if __name__ == '__main__':
+
+if __name__ == "__main__":
```

Comment:
BTW, `__main__` is actually not required if you are using pytest to trigger the tests.

Reply:
Removed.